src/back/node/src/api/list.cpp

```diff
@@ -0,0 +1,43 @@
+#include "list.hpp"
+#include "../model/node_serialize.hpp"
+#include "../service/service.hpp"
+#include <shared/headers.hpp>
+#include <shared/errors.hpp>
+#include <shared/errors_catchit.hpp>
+#include <shared/paging.hpp>
+#include <shared/paging_serialize.hpp>
+#include <shared/schemas.hpp>
+#include <shared/parse/uuid.hpp>
+
+namespace svetit::node::handlers {
+
+List::List(
+	const components::ComponentConfig& conf,
+	const components::ComponentContext& ctx)
+	: server::handlers::HttpHandlerJsonBase{conf, ctx}
+	, _s{ctx.FindComponent<Service>()}
+	, _mapHttpMethodToSchema{LoadSchemas(kName, _s.GetJSONSchemasPath())}
+{}
+
+formats::json::Value List::HandleRequestJsonThrow(
+	const server::http::HttpRequest& req,
+	const formats::json::Value& body,
+	server::request::RequestContext&) const
+{
+	formats::json::ValueBuilder res;
+
+	try {
+		const auto params = ValidateRequest(_mapHttpMethodToSchema, req, body);
+		const auto userId = params[headers::kUserId].As<std::string>();
+		const auto spaceId = params["spaceId"].As<boost::uuids::uuid>();
```

Comment:
spaceId неоткуда взяться если его никто не смапит
```suggestion
		const auto spaceId = params[std::string(headers::kSpaceId)].As<boost::uuids::uuid>();
```